### PR TITLE
[CBRD-25285] Add a new ClassLoader to avoid reloading classes that load JNI in a Java SP Server

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-25285

A new classloader (`StoredProcedureStaticClassLoader`) will be added to avoid reloading classes that load JNI. By `loadjava`'s `-j` option, classes are copied to `/java_static` directory and the classloader will try to find classes in the path.
The classloader does not check whether any file is modified in the path, Also is not be re-initialized like `StoredProcedureClassLoader`. To update overwritten classes in `/java_static`, restarting Java SP server(`cub_javasp`) is required.

### CODE REVIEW

**Review by beyondykk9 — APPROVED** [review]

File: src/executables/loadjava.cpp [hunk 1]
```diff
@@ -64,13 +70,14 @@ parse_argument (int argc, char *argv[])
...
     {"overwrite", 0, 0, 'y'},
+    {"jni", 0, 0, 'j'},
     {0, 0, 0, 0}
   };
 
   while (1)
     {
       int option_index = 0;
-      int option_key = getopt_long (argc, argv, "yh", loadjava_option, &option_index);
+      int option_key = getopt_long (argc, argv, "y:jh", loadjava_option, &option_index);
```
Comment (ctshim) [inline src/executables/loadjava.cpp]: Why did you add ':'?
Using ':' means there are arguments.
Does the '-y' option have additional arguments?

**Review by ctshim — APPROVED** [review]

File: msg/de_DE.utf8/utils.msg [hunk 2]
```diff
@@ -914,7 +914,8 @@ $set 34 MSGCAT_UTIL_SET_LOADJAVA
 Usage: loadjava [OPTION] database-name java-class-file\n\
 \n\
 valid options:\n\
-  -y, --overwrite              Klassendatei, wenn vorhanden, überschreiben, Standard nein\n
+  -y, --overwrite              Klassendatei, wenn vorhanden, überschreiben, Standard nein\n\
+  -j, --jni                    add a Class or Jar to static loading for JNI\n
```
Comment (hyunikn) [inline msg/de_DE.utf8/utils.msg:918]: Class --> class

**Review by hyunikn — APPROVED** [review]

**Comment by hgryoo** [comment]
build-windows fails casued by compiling `cubrid-jdbc` with JDK 1.6 version.
The following error is shown.
```
EXEC : [javac] error : Source option 6 is no longer supported. Use 7 or later. [C:\Users\circleci\project\build\cubrid-jdbc\jdbc_build.vcxproj]
EXEC : [javac] error : Target option 6 is no longer supported. Use 7 or later. [C:\Users\circleci\project\build\cubrid-jdbc\jdbc_build.vcxproj]
```
It seems not related to this PR, I'm going to merge the PR.
